webapp/tests/karma/ts/services/telemetry.service.spec.ts

```diff
@@ -95,7 +95,8 @@ describe('TelemetryService', () => {
       info: sinon.stub(),
       put: sinon.stub(),
       get: sinon.stub(),
-      query: sinon.stub()
+      query: sinon.stub(),
+      allDocs: sinon.stub()
     };
     dbService = { get: () => metaDb };
```

Comment:
I think we need to expand the testing here, since this query to get settings is NOT issued against the metaDb, but against the medicDb.

Reply:
I have renamed the variable for these mocks from metaDb to simply db to make it clearer that they are generic mocks and not just for the metaDb.

Reply:
I think we should have two mocks, since it's imperative we do some queries against one database, and other queries against another.